code/src/WijmoProvider/Features/ColumnSort.ts

```diff
@@ -162,8 +162,25 @@ namespace WijmoProvider.Feature {
             this.setState(this._enabled);
         }
 
-        public clear(): void {
-            this._grid.provider.collectionView.sortDescriptions.clear();
+        /**
+         * Function that clears sort of grid
+         * @returns {*}  {OSFramework.OSStructure.ReturnMessage} Return Message containing the resulting code from sorting columns and the error message in case of failure
+         */
+        public clear(): OSFramework.OSStructure.ReturnMessage {
+            try {
+                this._grid.provider.collectionView.sortDescriptions.clear();
+                return {
+                    isSuccess: true,
+                    message: 'Success',
+                    code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+                };
+            } catch (error) {
+                return {
+                    isSuccess: false,
+                    message: 'Error',
+                    code: OSFramework.Enum.ErrorCodes.API_FailedClear
+                };
+            }
```

Comment:
In what situations do exceptions occur?

Reply:
When for instance the Grid ID doesn't exist (but will fail first on the method that calls this one). 
In this particular case, I believe that we're doing this to keep consistency and prevent any error thrown from this._grid.provider.collectionView.sortDescriptions.clear();
Don't have a good example for it.

Reply:
if we don't have a good example, or any case where a problem occurs, why is this a problem to start with?
I'm a little against using try/catch, if not really needed.

Reply:
Just chatted with Gabriel. Please proceed.